src/dune_engine/sandbox.ml

```diff
@@ -2,6 +2,11 @@ open Import
 
 let sandbox_dir = Path.Build.relative Path.Build.root ".sandbox"
 
+let async_sandboxing f =
```

Comment:
Should be called `maybe_async` instead? This function is not always async (and is doesn't by itself do any sandboxing).